src/api/genes.ts

```diff
@@ -11,11 +11,11 @@ export const search = async (
   // params
   const params = new URLSearchParams();
   if (query) params.set("q", query);
-
   if (species?.length) params.set("species", species.join(","));
```

Comment:
`species` param is not working:

![image](https://user-images.githubusercontent.com/9020496/110862510-60fde000-8274-11eb-95c5-cf12c09f37fe.png)

The webpage returns too many results compared to the query:

![image](https://user-images.githubusercontent.com/9020496/110862614-6fe49280-8274-11eb-9aca-1ec686802fb3.png)

Reply:
I am only getting 3 results on the interface? 

![image](https://user-images.githubusercontent.com/8326331/110863400-9e6f6700-828e-11eb-88ea-5e4df2b8d75e.png)

Reply:
Also if you open the console (f12) you can see the exact query the frontend is making:

![image](https://user-images.githubusercontent.com/8326331/110863448-b3e49100-828e-11eb-99a4-1d1db9f856cb.png)

Reply:
Also I noticed in the previous PR that a checkbox looked messed up on your computer. What OS and browser and versions are you using?

Reply:
I think I know why you are getting this "error". You probably activated multi-search, typed "HCRTR1" then hit enter, creating a new line, then switched back to single search (which keeps the new line until you type again).

Currently how this works is if it detects line breaks (`\n`) in the input, it does a multi-search with POST, and otherwise does it with GET, which return different results.

I will change it so that if the text box is expanded it is always using POST, and if it's a single line it's always using GET.

Reply:
Weird... It's working now that I refreshed the page

Reply:
I'm using Firefox 86.0 on Ubuntu 20.04.

Reply:
Probably a caching issue then. I run into problems like this all the time, especially with Chrome.

Reply:
@vincerubinetti I think there's still an issue with species filtering for multiline searches.

Reply:
There's a lot of extra results when using a batch search:

![Peek 2021-03-11 17-29](https://user-images.githubusercontent.com/9020496/110878068-6288d180-828f-11eb-8aa3-5e8b486a5f01.gif)

Reply:
It is showing exactly what is returned for the query:

`https://mygene.info/v3/query?q=HCRTR1&fields=all&size=100&scopes=_id%2Calias%2Csymbol%2Centrezgene%2Censembl.gene%2Cuniprot&always_list=alias%2Csymbol%2Censembl%2Cuniprot`

Reply:
![image](https://user-images.githubusercontent.com/8326331/110894115-4869e600-82c5-11eb-992e-9ea08f24e513.png)